plugin.json

```diff
@@ -107,7 +107,24 @@
                 "help_text": "Display subscription name in post when a subscription posts to a channel",
                 "placeholder": "",
                 "default": false
+            },
+            {
+                "key": "JiraAuthAppClientID",
+                "display_name": "Jira App client ID",
+                "type": "text",
+                "help_text": "Jira App Client ID given when you create the app on JIRA. You can get it on Settings -> Authenticathion credentials",
+                "placeholder": "",
+                "default": ""
```

Comment:
If we want to configure multiple Jira Cloud instances, we will need config settings per Jira instance. Because of this, these settings will need to be stored in the KV store when setting up the instance. Does this make sense @javaguirre?